ios-drag-drop.js

```diff
@@ -1,12 +1,15 @@
 (function(doc) {
-function _exposeIosHtml5DragDropShim(config) {
 
+  
+function _exposeIosHtml5DragDropShim(config) {
+  var shimConfig;
   log = noop; // noOp, remove this line to enable debugging
 
   var coordinateSystemForElementFromPoint;
 
   function main() {
     config = config || {};
+    shimConfig = config;
```

Comment:
What's the purpose of this `shimConfig` variable? Looks to me that we could just check `config` directly like we do for other options.